tools/clang/tools/dxcompiler/dxclinker.cpp

```diff
@@ -184,7 +184,7 @@ HRESULT STDMETHODCALLTYPE DxcLinker::Link(
     CComPtr<IDxcBlob> pOutputBlob;
     CComPtr<AbstractMemoryStream> pDiagStream;
 
-    IFT(CoGetMalloc(1, &pMalloc));
+    IFT(DxcCoGetMalloc(1, &pMalloc));
```

Comment:
I think we can get rid of this, and just use `DxcGetThreadMallocNoRef()` in place of pMalloc below.

Reply:
Done.